test/e2e_node/eviction_test.go

```diff
@@ -395,6 +395,26 @@ var _ = SIGDescribe("PriorityMemoryEvictionOrdering", framework.WithSlow(), fram
 			{
 				evictionPriority: 1,
 				pod:              getMemhogPod("high-priority-memory-hog-pod", "high-priority-memory-hog", v1.ResourceRequirements{}),
+				prePodCreationModificationFunc: func(pod *v1.Pod) {
+					nodeList, err := f.ClientSet.CoreV1().Nodes().List(context.Background(), metav1.ListOptions{})
+					framework.ExpectNoError(err)
+					gomega.Expect(nodeList.Items).To(gomega.HaveLen(1))
```

Comment:
should we manage this in this anonymous function (and if so, should it be a `GinkgoHelper()` or should we want to check the assertion in the call site?